vms/avm/environment_test.go

```diff
@@ -313,7 +277,7 @@ func buildGenesisTestWithArgs(tb testing.TB, args *BuildGenesisArgs) []byte {
 	return b
 }
 
-func newTx(tb testing.TB, genesisBytes []byte, vm *VM, assetName string) *txs.Tx {
+func newTx(tb testing.TB, genesisBytes []byte, vm *VM, assetName string, chainID ids.ID) *txs.Tx {
```

Comment:
nit: consider passing env here, where genesisBytes and chainID comes from.

Reply:
reverted this change